libs/components/layout/src/lib/modules/back-to-top/back-to-top.component.spec.ts

```diff
@@ -92,6 +92,7 @@ describe('back to top component', () => {
       await fixture.whenStable();
       fixture.componentInstance.hideElement = true;
       fixture.detectChanges();
+      await fixture.whenStable();
```

Comment:
A note: this was removed in @Blackbaud-CoreyArcher's strict mode for back to top. However, I was having the test fail randomly without so I added it back.